app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -60,15 +65,19 @@ protected void onCreate(Bundle savedInstanceState) {
 
         fAuth = FirebaseAuth.getInstance();
         fStore = FirebaseFirestore.getInstance();
-        user = fAuth.getCurrentUser();
-        if (user != null) {
-            userId = user.getUid();
+
+        userManager = new FirestoreUserManager(fStore, FirestoreUserManager.USERS_COLLECTION, new UserSerializer());
```

Comment:
Where do you actually make use of this instance in your code (to implement the Firebase Firestore abstraction for the user?)

Reply:
For the time being, I refactored this code without making use of userManager.

For the next sprint, I'll ask Kepler and look at your code to see how we can abstract away the Firebase Firestore calls. I believe that there is more abstraction to be done here